docs/management_api.md

```diff
@@ -22,7 +22,17 @@ Similar to the [Inference API](inference_api.md), the Management API provides a
   * a URI using the HTTP(s) protocol. TorchServe can download .mar files from the Internet.
 * `model_name` - the name of the model; this name will be used as {model_name} in other APIs as part of the path. If this parameter is not present, `modelName` in MANIFEST.json will be used.
 * `handler` - the inference handler entry-point. This value will override `handler` in MANIFEST.json if present. **NOTE: Make sure that the given `handler` is in the `PYTHONPATH`. The format of handler is `module_name:method_name`.**
-* runtime - the runtime for the model custom service code. This value will override runtime in MANIFEST.json if present. The default value is `PYTHON`.
+* `torch_api_type` - the Torch API type to be used by the handler. This value will override `torchAPIType` in MANIFEST.json. If not provided, `torchAPIType` from MANIFEST.json will be used.
```

Comment:
We can have a different property name instead of 'torchAPIType'.  Suggestions are welcome.